pkg/monitoring/metrics/virt-handler/migrationstats_collector.go

```diff
@@ -53,41 +68,57 @@ var (
 	)
 )
 
-type migrationMetrics struct{}
+func setupMigrationStatsCollector(vmiInformer cache.SharedIndexInformer) error {
+	var err error
+	migrationdomainstatsHandler, err = migrationdomainstats.NewHandler(vmiInformer)
```

Comment:
What is the `migrationdomainstatsHandler`, is it needed?

Reply:
is used by `func migrationStatsCollectorCallback`, the callback executed by the Prometheus request, to fetch the collected metrics from the vmi migration